### ISSUE TITLE
Crash due to MediaPlayer state

### ISSUE THREAD

**Issue by AnwarShahriar** [hushed/WaveFormViewDemo#3]
```
java.lang.IllegalStateException
        at android.media.MediaPlayer.isPlaying(Native Method)
        at space.siy.waveformview.FixedWaveFormPlayer.isPlaying(FixedWaveFormPlayer.kt:181)
        at space.siy.waveformview.FixedWaveFormPlayer.stop(FixedWaveFormPlayer.kt:129)
        at space.siy.waveformview.FixedWaveFormPlayer.stop(FixedWaveFormPlayer.kt:123)
        at com.hushed.base.number.messaging.viewholders.AudioMessageViewHolder.stopAudio(AudioMessageViewHolder.java:98)
        at com.hushed.base.number.messaging.ConversationsMessageAdapter.onViewDetachedFromWindow(ConversationsMessageAdapter.java:234)
        at com.hushed.base.number.messaging.ConversationsMessageAdapter.onViewDetachedFromWindow(ConversationsMessageAdapter.java:26)
        at androidx.recyclerview.widget.RecyclerView.dispatchChildDetached(RecyclerView.java:7404)
        at androidx.recyclerview.widget.RecyclerView$5.removeViewAt(RecyclerView.java:870)
        at androidx.recyclerview.widget.ChildHelper.removeViewIfHidden(ChildHelper.java:386)
        at androidx.recyclerview.widget.RecyclerView.removeAnimatingView(RecyclerView.java:1450)
        at androidx.recyclerview.widget.RecyclerView$ItemAnimatorRestoreListener.onAnimationFinished(RecyclerView.java:12573)
        at androidx.recyclerview.widget.RecyclerView$ItemAnimator.dispatchAnimationFinished(RecyclerView.java:13073)
        at androidx.recyclerview.widget.SimpleItemAnimator.dispatchChangeFinished(SimpleItemAnimator.java:317)
        at androidx.recyclerview.widget.DefaultItemAnimator$7.onAnimationEnd(DefaultItemAnimator.java:370)
        at android.view.ViewPropertyAnimator$AnimatorEventListener.onAnimationEnd(ViewPropertyAnimator.java:1112)
        at android.animation.Animator$AnimatorListener.onAnimationEnd(Animator.java:554)
        at android.animation.ValueAnimator.endAnimation(ValueAnimator.java:1242)
        at android.animation.ValueAnimator.doAnimationFrame(ValueAnimator.java:1484)
        at android.animation.AnimationHandler.doAnimationFrame(AnimationHandler.java:146)
        at android.animation.AnimationHandler.access$100(AnimationHandler.java:37)
        at android.animation.AnimationHandler$1.doFrame(AnimationHandler.java:54)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:964)
        at android.view.Choreographer.doCallbacks(Choreographer.java:790)
        at android.view.Choreographer.doFrame(Choreographer.java:721)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:951)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7116)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:925)
```

**Comment by AnwarShahriar** [comment]
This can happen if `MediaPlayer` is not initialized. Might need to store the initialized state or reset the `MediaPlayer` in a `try/catch` block in `stop()/pause()/start()`.

**Comment by AnwarShahriar** [comment]
A fix is added to PR #11. Reopen this ticket if it happens again.
